wrapanapi/rest_client.py

```diff
@@ -116,6 +119,7 @@ def delete_status(self, path, headers=None):
         return r.ok
 
     def raw_get(self, path, headers=None, params=None):
+        self._logger.info('GET %s;', path)
```

Comment:
i believe the log level should be debug for that detail level

Reply:
Agreed